Adds Pricing & Pull Rate Limiting FAQs page (#11840)
docker/docker.github.io: docker-hub/billing/pricing-pull-rate-limit-faqs.md

* NEW Pro plan for individuals with unlimited private repos, unlimited public repos, and unlimited collaborators starting at $5/month with an Annual subscription.
* NEW Team plan for development teams with unlimited private repos and unlimited public repos starting at $25 per month for the first 5 users and $7 per month for each user thereafter with an Annual subscription. The Team plan offers advanced collaboration and management tools, including organization and team management with role-based access controls.

### Are prices listed in US dollars?

All prices reflect US dollars.

### How does Team pricing work?

Team starts at $25 per month for the first 5 users and $7 per month for each user thereafter with an Annual subscription.

### How will the new pricing plan impact existing Docker Hub customers?

Legacy customers have until January 31, 2021 to switch to the new pricing plans. For more information on how to move to the new pricing plans, view the [Resource Consumption Updates FAQ](https://www.docker.com/pricing/resource-consumption-updates).

### How can I compare which features are in each plan?

> You can see pricing and a full list of features for each product at [https://www.docker.com/pricing](https://www.docker.com/pricing).

Reviewer: ```suggestion
You can see pricing and a full list of features for each product at [Docker pricing](https://www.docker.com/pricing){:target="_blank" rel="noopener" class="_"}.
```

> You can see pricing and a full list of features for each product at [https://www.docker.com/pricing](https://www.docker.com/pricing).

Reviewer: ```suggestion
You can see pricing and a full list of features for each product at [Docker Pricing](https://www.docker.com/pricing){: target="blank" rel="noopener" class=""}.
```

### What is the difference between the legacy plans and the newly announced plans?

The legacy plans were based on a private repository/parallel autobuild pricing model. The new Pro and Team plans are now based on a per-seat pricing model. Both Pro and Team offer unlimited private repos. The Free plan offers unlimited public repositories at no cost per month.

### If I am an existing paid Docker Hub customer, when do I need to change my plan?

If you are an existing Docker subscriber you have until January 31st 2021 to move to the new pricing plans mentioned above. You can convert to either a Monthly plan or an Annual plan.

### If I am an existing subscriber but I don’t do anything by the due date, what will happen?

If no action is taken by the date, Docker will automatically upgrade you to an equivalent plan.

### Will my price per month increase or decrease?

Depending on your configuration you may find it more economical to move to one of the new pricing plans available. For Teams, the key factor affecting a price increase or decrease is the total number of seats you need to support your organization.

### What are the differences between the Team and Free Team plans?

> For details on the differences for each plan option, please see [https://docs.docker.com/docker-hub/billing/](https://docs.docker.com/docker-hub/billing/).

Reviewer: ```suggestion
For details on the differences for each plan option, please see [Docker Billing](index.md).
```

### Do collaborator limits differ between the Free and Pro plans?

The Free plan includes unlimited collaborators for public repositories and 0 collaborators for private repositories. Pro includes unlimited collaborators for public repositories and 1 unique collaborator for private repositories. Note that a collaborator on a private repository within a Pro plan can only be used with a service account.

> For details on the differences between plans, see [https://www.docker.com/pricing](https://www.docker.com/pricing).

Reviewer: ```suggestion
For details on the differences between plans, see [Docker Pricing](https://www.docker.com/pricing){: target="blank" rel="noopener" class=""}.
```

> For more information on service accounts visit our [docs](/docker-hub/repos/#service-accounts) page.

Reviewer: ```suggestion
For more information, see [Service accounts](/docker-hub/repos/#service-accounts) page.
```

### How can I create a new Docker Hub account?

> You can create a new account at [https://www.docker.com/pricing](https://www.docker.com/pricing) where you can choose a plan for Individuals or a plan for Teams.

Reviewer: ```suggestion
You can create a new account at [Docker Pricing](https://www.docker.com/pricing){: target="blank" rel="noopener" class=""} where you can choose a plan for Individuals or a plan for Teams.
```

### How do I upgrade to a Team plan from a legacy organization plan?

Upgrading your legacy plan to a Team plan offers you unlimited private repos, unlimited teams, and 3 parallel builds. For information on how to upgrade to a Team plan from a legacy (per-repository) plan, see [Upgrade to a Team plan](/docker-hub/billing/upgrade/#upgrade-to-a-team-plan).

### How do downgrades from a Pro or Team plan work?

When you downgrade your Pro or Team plan, changes are applied at the end of your billing cycle. For example, if you are currently on a Team plan which is billed on the 8th of every month and you choose to downgrade to a Free Team plan on the 15th, your Team plan will be active until 7th of the following month. You will be transferred to a Free Team plan on the 8th of the following month. Please refer to the next two questions below for instructions on how to downgrade your account.

### How do I downgrade from a Team plan to a Free Team plan?

> Before you downgrade to a Free plan, ensure that your organization details are updated to reflect features available in the Free plan. For example, you may need to reduce the number of team members and convert any private repositories to public repositories. For information on what’s included in the Free plan, see [https://docs.docker.com/docker-hub/billing/](/docker-hub/billing/).

Reviewer: Should we link this to the www pricing page instead of docs for details on what's included in the Free plan?

For information on how to downgrade from a Team plan to a Free plan, see [Downgrade from Team to a Free plan](/docker-hub/billing/downgrade/#downgrade-from-team-to-a-free-plan).

### How do I downgrade from Pro to a Free plan?

> Before you downgrade to a Free plan, ensure that your account organization details are updated to reflect features available in the Free plan. For example, you may need to convert any private repositories to public repositories. For information on what’s included in the Free plan, see [https://docs.docker.com/docker-hub/billing/](/docker-hub/billing/).

Reviewer: Should we link to www instead?

### How do I add or remove paid seats for my team members?

For information on how to add paid seats to a monthly plan for your organization, see [Add seats to a monthly plan](/docker-hub/billing/add-seats/).

For information on how to remove paid seats from a monthly plan for your organization, see [Remove seats from a monthly plan](/docker-hub/billing/remove-seats/).

### I want to run an automated agent that makes container requests on behalf of my organization. Which license do I need?

Automated agents or service accounts that make container image requests of Docker Hub must be licensed under a Docker Team subscription.

### How do I add a member to a team in my organization?

For information on how to add a member to a team, see [Add a member to a team](/docker-hub/orgs/#add-a-member-to-a-team).

### Do you offer annual subscriptions?

Yes! Both the Pro and Team plans are available via annual subscriptions and include a discount from the monthly subscription price.

> You can view the annual subscription pricing for each product at [https://www.docker.com/pricing](https://www.docker.com/pricing).

Reviewer: ```suggestion
You can view the annual subscription pricing for each product at [Docker Pricing](https://www.docker.com/pricing){: target="blank" rel="noopener" class=""}.
```

### Are legacy plans eligible for annual pricing?

No, you must move to the new Pro or Team plans to take advantage of the Annual pricing.

### Can Docker Hub continue to be used for open source projects?

Yes, Docker will continue to offer a Free plan with unlimited public repositories and unlimited collaborators for public repositories at no cost per month. Docker is committed to supporting the broader open source communities.

> For questions about open source projects and pricing please visit the [Open Source Community Application](https://www.docker.com/community/open-source/application).

Reviewer: ```suggestion
For questions about open source projects and pricing please visit the [Open Source Community Application](https://www.docker.com/community/open-source/application){: target="blank" rel="noopener" class=""}.
```

### I have more questions regarding the new pricing, who can I reach out to?

If you have any questions about how the new seat-based pricing impacts you that are not covered in the FAQ, please reach out and a member of the Docker team will get back to you.

## Pull Rate Limiting FAQs

### What are the Docker Terms of Service?

> The Docker Terms of Service is an agreement between you and Docker that governs your use of Docker products and services. Click on the link to view the full [Docker Terms of Service](https://www.docker.com/legal).

Reviewer: ```suggestion
The Docker Terms of Service is an agreement between you and Docker that governs your use of Docker products and services. Click on the link to view the full [Docker Terms of Service](https://www.docker.com/legal){: target="blank" rel="noopener" class=""}.
```

Reviewer: Could you also add `target=blank` to any external URLs in this topic?

### When do the Docker Terms of Service take effect?

The updates to the [Docker Terms of Service](https://www.docker.com/legal) are effective immediately.

### What changes is Docker making to the Terms of Service?

The most notable changes are in Section 2.5. To see all of the changes, we recommend you read the full [Docker Terms of Service](https://www.docker.com/legal).

### I read that images are going to expire on November 1? What is going on here?

A planned policy for image expiration enforcement for inactive images for free accounts was announced to take effect November 1, 2020. After hearing feedback from customers, this policy has been placed on hold until mid 2021. More information is available in [this blog post](https://www.docker.com/blog/docker-hub-image-retention-policy-delayed-and-subscription-updates/).

### What are the rate limits for pulling Docker images from the Docker Hub Registry?

Rate limits for Docker image pulls are based on the account type of the user requesting the image - not the account type of the image’s owner. These are defined on the [pricing](https://www.docker.com/pricing) page.

> The highest entitlement a user has, based on their personal account and any orgs they belong to, will be used. Unauthenticated pull requests are “anonymous” and will be rate limited based on IP address rather than user ID. For more information on authenticating image pulls, please see this [docs page](https://docs.docker.com/docker-hub/download-rate-limit/#how-do-i-authenticate-pull-requests).

Reviewer: Could you use relative path instead to point to the Download rate limits topic?

### Are image layers counted?

No. Because we are limiting on manifest requests, the number of layers (blob requests) related to a pull is unlimited at this time. This is a change from our previous policy based on community feedback in order to be more user-friendly, so users do not need to count layers on each image they may be using.

### Are anonymous (unauthenticated) pulls rate-limited based on IP address?

Yes. Pull rates are limited based on individual IP address (e.g., for anonymous users: 100 pulls per 6 hours per IP address). See [pricing](https://www.docker.com/pricing) for more details.

### Are pulls from logged-in accounts rate-limited based on IP address?

No, limits on pull requests that are [authenticated with an account](https://docs.docker.com/docker-hub/download-rate-limit/#how-do-i-authenticate-pull-requests) are based on that account and not on the IP. Free accounts are limited to 200 pulls per 6 hour period. Paid accounts are subject to fair usage as outlined in Docker [pricing](https://www.docker.com/pricing).

### Will I be rate-limited if I’m logged in but someone else on my IP hits the anonymous limit?

> No, users who are logged in and [authenticating their pull requests](https://docs.docker.com/docker-hub/download-rate-limit/#how-do-i-authenticate-pull-requests) will be limited based on their user account only. If an anonymous user on your IP is limited, you will not be affected as long as your requests are authenticated and you have not hit your own limit.

Reviewer: Could you use relative path intead?

### Will Docker offer dedicated plans for open source projects?

Yes, as part of Docker’s commitment to the open source community, we have created a new program to support free use of Docker in open source projects. [See this blog post](https://www.docker.com/blog/expanded-support-for-open-source-software-projects/) for more information.

To apply for an open source plan, complete our application at:

[https://www.docker.com/community/open-source/application](https://www.docker.com/community/open-source/application).

For more information view our blogs on [inactive image retention](https://www.docker.com/blog/scaling-dockers-business-to-serve-millions-more-developers-storage) and [pull rate limits](https://www.docker.com/blog/scaling-docker-to-serve-millions-more-developers-network-egress).

Or check out: [https://www.docker.com/pricing](https://www.docker.com/pricing)

> <script type="application/ld+json">

Reviewer: @jhratliff Do we need this script in docs? Wondering if this got added by mistake?

Author: @usha-mandya - the script is what exposes the page content as structured data to the search crawlers. Based on the JSON data found on the page, the content will now be available in rich search features.

Reviewer 2: Looks like the script is largely duplicating the whole page; wondering how we'll keep the script in sync with the actual page-content.

Wondering if there's a Jekyll plugin that allows us to annotate headings/sections to convert these to `ld+json`.

Or do you know if there's a way for this rich metadata to be constructed by Google / search engines from the actual page content (adding some markup/annotation to sections)?

Author: I tend to build these out as content fields in CMS templates so we only enter the data once in a DB structure. The template would then format the content for consumption by humans and crawlers. I haven't yet had a chance to dig into the Jekyll system to see what kind of data structures are available for this kind of use (I'm only seeing the static, hand-crafted content thus far).
